spec/ffmpeg/movie_spec.rb

```diff
@@ -216,15 +221,18 @@ module FFMPEG
       end
 
       context "given a file with non supported audio" do
-        before(:all) do
-          fake_stdout = StringIO.new(File.read("#{fixture_path}/outputs/file_with_non_supported_audio_stdout.txt"))
-          fake_stderr = StringIO.new(File.read("#{fixture_path}/outputs/file_with_non_supported_audio_stderr.txt"))
-          Open3.stub(:popen3).and_yield(nil, fake_stdout, fake_stderr)
+        before(:each) do
+          fake_stdout = File.read("#{fixture_path}/outputs/file_with_non_supported_audio_stdout.txt")
+          fake_stderr = File.read("#{fixture_path}/outputs/file_with_non_supported_audio_stderr.txt")
+          spawn_double = double(:out => fake_stdout, :err => fake_stderr)
+          puts 'ERROR'
+          puts spawn_double.err
+          POSIX::Spawn::Child.stub(:new).and_return(spawn_double)
           @movie = Movie.new(__FILE__)
         end
 
-        it "should not be valid" do
-          @movie.should_not be_valid
+        it "should be valid" do
+          @movie.should be_valid
```

Comment:
hmm how come this spec changed?

Reply:
AFAICT we changed this behaviour with https://github.com/mattcook/rlovelett-ffmpeg/pull/8 but didn't update the spec. It's only invalid if _both_ video and audio are unsupported.